svg-2-android-vector/src/main/java/com/quittle/svg2androidvector/Svg2AndroidVectorPlugin.java

```diff
@@ -32,12 +39,14 @@
     private static final String SVG_FILE_EXTENSION = ".svg";
     private static final String XML_FILE_EXTENSION = ".xml";
     private static final String BUILD_DIR_RESOURCE_NAME = "android-vector-resources";
-    private static final String EARLY_ANDROID_TASK_NAME = "preBuild";
     // TODO: See if the drawable folder may be used instead of raw.
     private static final String SVG_FILTER_PATTERN =
             String.format("%s/**/*%s", ANDROID_RESOURCES_DIR_NAME_RAW, SVG_FILE_EXTENSION);
     private static final String CONVERSION_TASK_NAME_FORMAT = "ConvertSvgToXml-%s";
 
+    private static final Logger logger =
+            Logging.getLogger(Svg2AndroidVectorPlugin.class);
```

Comment:
I'd prefer grabbing the logger with `project.getLogger()` for now. One day I'll create a plugin-specific logger

Reply:
ah, didn't realize that was a thing :)